juju/client/gocookies.py

```diff
@@ -104,5 +104,23 @@ def py_to_go_cookie(py_cookie):
         unix_time = datetime.datetime.fromtimestamp(py_cookie.expires)
         # Note: fromtimestamp bizarrely produces a time without
         # a time zone, so we need to use accept_naive.
-        go_cookie["Expires"] = pyrfc3339.generate(unix_time, accept_naive=True)
+        go_cookie["Expires"] = generate_rfc3339_from_unix_time(unix_time)
```

Comment:
I see that there's a solid-looking set of tests in `tests/unit/test_gocookies.py`.
Could you clarify why the custom parses function was needed, what specific inputs could not be parsed?

Reply:
ciso8601 does not have a function to generate a rfc3339 timestamp from a datetime, as this one of the pyrfc3339 library, so I implemented my own.

Reply:
I wonder if `datetime.isoformat()` would be enough.
I'll need someone who knows about go-style cookies and why are those important, if Juju relies on those in some way.